releases/nightly-build/compose-files/docker-compose-nexus-redis.yml

```diff
@@ -266,7 +266,7 @@ services:
       <<: *redis5-variables
     command: |
       /bin/sh -c "
-      until [ -r $${REDIS5_PASSWORD_PATHNAME} ]; do sleep 1; done
+      until [ -r $${REDIS5_PASSWORD_PATHNAME} ] && [ -s $${REDIS5_PASSWORD_PATHNAME} ]; do sleep 1; done
```

Comment:
I tested this change and still seeing same issue on both my Windows and Linux environments. I will ask others with similar systems like @rsdmike. @jim-wang-intel and @bnevis-i to test.

Reply:
Then I suggest I put a 10 second sleep in the entry script and eliminate the intermediate password file after Geneva release.

Reply:
@andresrinivasan , I changed the `do sleep 1` to `do sleep 10`. Still getting the error.

Reply:
IMO, putting sleep here may not help as the other container still starts up; it will help if support-logging has the waiting mechanism for redis ready signal

Reply:
New strategy after debugging session with @lenny-intel and @bnevis-i. 

> The first connection refused is a time issue w/o a retry loop for the lazy connection to redis. Second is the old password is still on disk when Redis starts, so it gets the old password and the services pickup the new password. The volume prune does clean the direct mount volumes.